datashuttle/datashuttle.py

```diff
@@ -174,37 +159,37 @@ def make_sub_dir(
 
     def upload_data(
         self,
-        experiment_type: str,
         sub_names: Union[str, list],
         ses_names: Union[str, list],
-        preview: bool = False,
+        experiment_type: str = "all",
+        dry_run: bool = False,
     ):
         """
         Upload data from a local machine to the remote project
         directory. In the case that a file / directory exists on
         the remote and local, the local will not be overwritten
         even if the remote file is an older version.
 
-        :param experiment_type: see make_sub_dir()
         :param sub_names: a list of sub names as accepted in make_sub_dir().
                           "all" will search for all sub- directories in the
                           data type directory to upload.
         :param ses_names: a list of ses names as accepted in make_sub_dir().
                           "all" will search each sub- directory for
                           ses- directories and upload all.
-        :param preview: perform a dry-run of upload, to see which files
+        :param dry_run: perform a dry-run of upload, to see which files
                         are moved.
+        :param experiment_type: see make_sub_dir()
```

Comment:
Much better for the user to be able to read the experiment_types here. You can still direct to make_sub_dir() for details